crates/autopilot/src/infra/blockchain/mod.rs

```diff
@@ -13,7 +13,7 @@ use {
 };
 
 pub mod contracts;
-pub mod dto;
+mod dto;
```

Comment:
nit: I'm not sure dto is the right name for this (we are basically creating a model for infra vs. domain representations, but not a data transfer object). How do we do infra -> domain conversions in other places of the codebase? 

Maybe we can just lift this code up into the main mod?

Reply:
We have a very similar module for persistence: https://github.com/cowprotocol/services/tree/main/crates/autopilot/src/infra/persistence/dto

I think `dto` was used to indicate that there is some conversion of data between layers happening. IMO, we should have a separate module for each section of persistence and each conversion to be put together with the functions called in that module, but we decided to put all persistence functions in a single file until it becomes large.

Reply:
> We have a very similar module for persistence:

Well, you wrote this one very recently, I was more wondering on how things are done in the driver, where we had more fundamental discussions of how to adopt DDD in our codebase. There, dto is only used for the API objects we put in the wire to talk between core service components, which is not the case here.

Maybe this is a case for boundary::transaction and a `to_domain` function a la https://github.com/cowprotocol/services/blob/4a6f155004470414591b3b3e1c86118a8f1176d5/crates/driver/src/boundary/liquidity/balancer/v2/weighted.rs#L17

?

Reply:
Resolved as a free function under `mod.rs` which is still very small for blockchain so I guess it's fine.